nixos/packages.tt

```diff
@@ -33,7 +33,7 @@
     <tr>
       <th>Install command:</th>
       <td>
-        <tt><span class="muted">$</span> nix-env -iA nixos.<span class='attrname'></span></tt> <em class='muted'>(NixOS channel)</em>
+        <tt><span class="command">nix-env -iA nixos.<span class='attrname'></span></span></tt> <em class='muted'>(NixOS channel)</em>
```

Comment:
Small style nit, even though double-quotes are the general "style of the web", _this_ specific file seems to use single-quotes for all HTML tag attributes. For consistence, could you make this:

```html
- <span class="command">
+ <span class='command'>
```

... that, or make a separate PR to update the rest of this file to use double quotes (which is my _personal_ recommendation) =P